apps/docs/src/app/documentation/core-helpers/example-background/example-background.component.ts

```diff
@@ -3,7 +3,10 @@ import { Component, Input } from '@angular/core';
 @Component({
     selector: 'background-toggle',
     template: `
-        <fd-toggle style="margin-bottom: 18px" [size]="'xs'" (checkedChange)="onChange()">Toggle background</fd-toggle>
+    <label fd-form-label>
+        Toggle background
+    </label>
+    <fd-toggle style="margin-bottom: 18px" (checkedChange)="onChange()"></fd-toggle>
```

Comment:
why we put this styling (`style="margin-bottom: 18px"`) here?

Reply:
From what I can tell it was put because our components are self-contained (for styles) and don't have any margins. This way our containers that hold the examples which have the two toggles at the top have a margin bottom